src/scripts/tweaks.json

```
@@ -51,6 +51,11 @@
       "type": "checkbox",
       "label": "Hide the \"Now, where were we?\" button",
       "default": false
+    },
+    "caught_up_line": {
+      "type": "checkbox",
+      "label": "Turn \"You're caught up\" into a separating line",
+      "default": false
     }
   }
-}
+}
```

Nitpick: newline at end of file is preferred

```suggestion
}

```

I recommend setting up EditorConfig!